editor/src/clj/editor/collection.clj

```diff
@@ -216,9 +237,7 @@
   (output node-outline outline/OutlineData :cached produce-go-outline)
   (output ddf-message g/Any :abstract)
   (output node-outline-extras g/Any (g/constantly {}))
-  (output build-resource resource/Resource :abstract)
-  (output build-targets g/Any produce-go-build-targets)
-  (output build-error g/Err (g/constantly nil))
+  (output build-targets g/Any :abstract)
```

Comment:
Instead of trying to share code in the base node, we now have separate `build-targets` implementations instead.